<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fourGfacing</class>
 <widget class="QDialog" name="fourGfacing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <property name="text">
    <string>Data Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <property name="text">
    <string>Time slot:</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd HH:mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <property name="text">
    <string>to</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd HH:mm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_statu">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>60</y>
     <width>87</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>NO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OFF</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>819</x>
     <y>110</y>
     <width>271</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <property name="title">
    <string>INFORMATION</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_id">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
    </property>
    <property name="text">
     <string>install time:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_inspection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
    </property>
    <property name="text">
     <string>inspection point number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
    </property>
    <property name="text">
     <string>Latitude and lobgit:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_lo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_la">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>427</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
    </property>
    <property name="text">
     <string>change finish</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>811</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string>chart</string>
   </property>
   <widget class="QChartView" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>811</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>line chart</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bar chart</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <property name="text">
    <string>Start collecting</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
